magprime/models.py

```diff
@@ -12,6 +12,24 @@ def invalid_notification(self):
             except:
                 log.error('unable to send invalid email')
 
+    @cost_property
+    def child_discount(self):
+        if 'val' in self.age_group_conf and self.age_group_conf['val'] == c.UNDER_13:
+            return (c.BADGE_PRICE / 2) * -1
```

Comment:
i just ran this and it can return a decimal (like $55/2 = $27.50).  do we want to do any kind of rounding, or is this OK?  mostly asking because we haven't had non-int pricing before.  in theory it should be OK